wd_helper.au3

```diff
@@ -88,7 +88,8 @@ Global Enum _
 ;                  - $_WD_ERROR_Timeout
 ; Author ........: Danp2
 ; Modified ......: mLipok
-; Remarks .......: For list of $sFeatures take a look in the following link
+; Remarks .......: For list of $sFeatures take a look in the following link.
+;                  The value returned by _WD_LastHTTPResult() is changed by this function
```

Comment:
I don't like having this in the remarks for each function that affects _WD_LastHTTPResult. I would like to come up with a better alternative. A couple of ideas -- 1) list _WD_LastHTTPResult as a related function 2) some for of footnote

Reply:
1 is Ok for me 
Will change this

Reply:
Done